client/rpcserver/handlers.go

```diff
@@ -167,15 +167,15 @@ func handleNewWallet(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 		resErr := msgjson.NewError(msgjson.RPCCreateWalletError, errMsg)
 		return createResponse(newWalletRoute, nil, resErr)
 	}
-	s.notifyWalletUpdate(form.assetID)
+	s.wsServer.NotifyWalletUpdate(form.assetID)
```

Comment:
Oh, this just answered why a shared websocket hub/server instance....  I'm still thinking of ways to avoid this though.

Reply:
Another approach would be to leave it to `Core` to notify.  `WebServer` already relays from core's notification feed: https://github.com/decred/dcrdex/blob/a2ef2bee6f21925d9fee3037ee3fc53c25d7fcd7/client/webserver/webserver.go#L377-L384

So if these core methods (CreateWallet, OpenWallet, CloseWallet) did the notification it would still make it to all subscriber.

Reply:
Now that I think this through a bit more, it's a bit backwards for the **consumer** of `Core` to send `WalletState` updates, while all other updates, such as `'order'`, `'balance'`, and `'update_wallet'` type notes originate within `Core`.  Note that these wallet state updates are even handled with their own route (`'notify'`) while the order/balance/feepayment ones from Core are sent for the `'notify'` route.

@buck54321 Thoughts about unifying the messages handling on frontend?  And on backend the way I described, always from `Core` if the intention is to have all feed subscribers get the info?

Reply:
Yeah. The notifications should all come from **core**. `NotifyWalletUpdate` should disappear, and we can send the `update_wallet` notification from `CreateWallet`, `OpenWallet`, and `CloseWallet` instead.

Reply:
@JoeGruffins Does the solution seem straightforward?  

Although, since this PR keeps this existing behavior, I think the notification refactoring _could_ be a subsequent PR.  Plus I have some hacking I'd like to do in these files.  :)  So if it looks unclear or hairy, and unless there's actually a regression I'm not seeing, perhaps we can move forward with the websocket package refactoring.  That said, it doesn't seem like a major change to make.

Reply:
I will leave this for now. I also have a few small tweaks I would like to make, but I will wait to see what you do. So, feel free to start fixing.